lib/api/v3/relations/relation_representer.rb

```diff
@@ -52,6 +52,18 @@ class RelationRepresenter < ::API::Decorators::Single
           }
         end
 
+        link :toType do
+          {
+            href: api_v3_paths.type(represented.to.type_id)
+          }
+        end
+
+        link :fromType do
+          {
+            href: api_v3_paths.type(represented.from.type_id)
+          }
+        end
```

Comment:
We're still using it here: https://github.com/opf/openproject/pull/5639/files#diff-d73e3377d1e7f10d8cdf091ce5cc523fR92

And since relations are tightly coupled to WP types in the frontend, we can use these links to make the relations tab much simpler without loading all work packages up front to do the sorting.